sparse/tests/test_core.py

```diff
@@ -121,8 +121,9 @@ def test_to_scipy_sparse():
     a = s.to_scipy_sparse()
     b = scipy.sparse.coo_matrix(s.todense())
 
-    assert_eq(a.data, b.data)
     assert_eq(a.todense(), b.todense())
+    assert_eq(a.data[np.lexsort((a.row, a.col))],
+              b.data[np.lexsort((b.row, b.col))])
```

Comment:
Just added a workaround, but I think this check is not necessary here.
(Do we want to preserve the order of the entries?)

Reply:
No, we do not want to preserve the order of entries. It's worth looking into why this test failed. I may take a look later today... I suspect it's because of out of order entries but I can't say much more.

Reply:
I looked into this. I'd say it's safe to remove this check, it was meant for when the entries were sorted. As long as `todense` is okay, we should be good.